deployments/apiv2/services/users/src/main.py

```diff
@@ -95,15 +97,16 @@ async def login(request: Request, details: Union[UserLogin, CustomerLogin]):
     else:
         account_type = "user"
         # suspended is for deactivated accounts and verified is for new users needing to verify through email
-        select_query = (
-            "SELECT password, id, data->'scope' AS scope "
-            "FROM users WHERE deleted_at IS NULL AND name=$1 AND customer_id=$2 AND suspended='f' AND verified='t'"
+        # select for service specific usage restrictions listed under the customer account
+        select_query = "SELECT password, id, data->'scope' AS scope FROM users WHERE deleted_at IS NULL AND name=$1 AND customer_id=$2 AND suspended='f' AND verified='t'"
+        select_query_params = (
+            details.username,
+            str(details.customer_id),
```

Comment:
Were these formatting changes necessary? the assignment to `select_query` was written the original way where it wraps multiple lines so it's more readable

Reply:
did the hooks not  do this? I don't remember if this was me or the formatting hooks, but can  switch it back yes

Reply:
Oh yeah, not sure if the pre-commit hooks would change this